qubesbuilder/plugins/source/scripts/get-and-verify-source

```diff
@@ -0,0 +1,392 @@
+                                             because only commits that have been reviewed are tagged.
+    --maintainer                             Allowed maintainer provided as KEYID assumed to be available as KEYID.asc
+                                             under provided --keys-dir directory. Can be used multiple times.
+"
+}
+
+print_headers() {
+    if [ -z "$GIT_OPTIONS" ]; then
+        GIT_INFOS="$GIT_URL $BRANCH"
+    else
+        GIT_INFOS="$GIT_URL $BRANCH (options: $GIT_OPTIONS)"
+    fi
+
+    echo "-> Updating sources for $COMPONENT..."
+    echo "--> Fetching from $GIT_INFOS..."
+}
+
+verify_git_obj() {
+    local content newsig_number
+    content=$(git -c gpg.program=gpg -c gpg.minTrustLevel=fully "verify-$1" --raw -- "$2" 2>&1 >/dev/null) &&
+        newsig_number=$(printf %s\\n "$content" | grep -c '^\[GNUPG:] NEWSIG') &&
+        [ "$newsig_number" = 1 ] && {
+        printf %s\\n "$content" |
+            grep '^\[GNUPG:] TRUST_\(FULLY\|ULTIMATE\) 0 pgp$' >/dev/null
+    }
+}
+
+exit_clean() {
+    local exit_code=$?
+    if [ $exit_code -gt 0 ]; then
+        # if verification failed, remove fetched content to make sure we'll not
+        # use it
+        if "$fresh_clone"; then
+            rm -rf "$REPO"
+        else
+            rm -f "$REPO/.git/FETCH_HEAD"
+        fi
+    fi
+    exit "$exit_code"
+}
+
+if ! OPTS=$(getopt -o '' -l help,git-baseurl:,git-prefix:,git-suffix:,git-url:,git-remote:,git-branch:,component:,clean:,fetch-only:,ignore-missing:,repo:,keys-dir:,keyring-dir-git:,insecure-skip-checking,less-secure-signed-commits-sufficient,maintainer: -n "$0" -- "$@"); then
+    echo "ERROR: Failed while parsing options."
+    exit 1
+fi
+
+eval set -- "$OPTS"
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+    --help) usage ;;
+    --git-baseurl)
+        GIT_BASEURL="$2"
+        shift
+        ;;
+    --git-prefix)
+        GIT_PREFIX="$2"
+        shift
+        ;;
+    --git-suffix)
+        GIT_SUFFIX="$2"
+        shift
+        ;;
+    --git-url)
+        GIT_URL="$2"
+        shift
+        ;;
+    --git-remote)
+        GIT_REMOTE="$2"
+        shift
+        ;;
+    --git-branch)
+        BRANCH="$2"
+        shift
+        ;;
+    --component)
+        COMPONENT="$2"
+        shift
+        ;;
+    --clean)
+        CLEAN="$2"
+        shift
+        ;;
+    --fetch-only)
+        FETCH_ONLY="$2"
+        shift
+        ;;
+    --ignore-missing)
+        IGNORE_MISSING="$2"
+        shift
+        ;;
+    --repo)
+        REPO="$2"
+        shift
+        ;;
+    --keys-dir)
+        KEYS_DIR="$2"
+        shift
+        ;;
+    --keyring-dir-git)
+        KEYRING_DIR_GIT="$2"
+        shift
+        ;;
+    --insecure-skip-checking)
+        INSECURE_SKIP_CHECKING=1
+        shift
+        ;;
+    --less-secure-signed-commits-sufficient)
+        LESS_SECURE_SIGNED_COMMITS_SUFFICIENT=1
+        shift
```

Comment:
```suggestion
```

Reply:
Already fixed. This is in `fetch` plugin now.